components/ContractTokens.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-card v-if="tokenBalancesLoaded" class="mx-auto my-12" max-width="500">
+    <v-card-title>Tokens</v-card-title>
+    <v-card-subtitle>{{ this.collateralAddress }}</v-card-subtitle>
+    <v-card-text>
+      <v-list-item>
+        <v-list-item-title> Collateral Balance </v-list-item-title>
+        <v-list-item-subtitle>{{ this.collateralTokens }}</v-list-item-subtitle>
+      </v-list-item>
+
+      <v-list-item>
+        <v-list-item-title> Long Balance </v-list-item-title>
+        <v-list-item-subtitle>{{ this.syntheticTokens.longBalance }}</v-list-item-subtitle>
+      </v-list-item>
+
+      <v-list-item>
+        <v-list-item-title> Short Balance </v-list-item-title>
+        <v-list-item-subtitle>{{ this.syntheticTokens.shortBalance }}</v-list-item-subtitle>
+      </v-list-item>
+    </v-card-text>
+
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Vue, Component, namespace, Prop } from "nuxt-property-decorator"
+import SyntheticTokenBalances from "@/store/contracts"
+import { LSPConfiguration } from "~/types"
+
+const addresses: Record<string, string> = require("@/addresses.json")
```

Comment:
same goes here on import vs require. I know we also have two aliases for our rootdir, this included by some boilerplate stuff that I used, but I feel like we should be consistent on it. We have tilde in most places but I don't mind changing to \@

Reply:
See above comment, I would like to stick to require for now just to avoid spending more time on this than I already have. But feel free to just adjust all the requires to imports at a later point. (It will probably take you 5mins, I am just getting all kinds of errors right now when I do the switch)